webpack.component.config.js

```diff
@@ -88,6 +95,25 @@ module.exports = {
       "Access-Control-Allow-Methods": "GET, POST, PUT, DELETE, PATCH, OPTIONS",
       "Access-Control-Allow-Headers":
         "X-Requested-With, content-type, Authorization",
+      // Pyodide - required for input and code interruption - needed on the host app
+      "Cross-Origin-Opener-Policy": "same-origin",
+      "Cross-Origin-Embedder-Policy": "require-corp",
```

Comment:
So are we just assuming that host pages will have to add these themselves for now, or am I not understanding?

Reply:
yea that's right. So on editor.raspberrypi.org we've already got these set 👍